util/fibers/uring_buf_pool.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024, Roman Gershman.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+#include "util/fibers/uring_buf_pool.h"
+
+#include <cstdint>
+#include <new>
+#include <optional>
+#include <system_error>
+
+#include "base/logging.h"
+#include "util/fibers/proactor_base.h"
+#include "util/fibers/uring_proactor.h"
+
+namespace util::fb2 {
+
+namespace {
+
+thread_local UringBufPool tl_buf_pool;
```

Comment:
Why not make it a uring proactor member?

Reply:
Actually, good idea